Formula/lua.rb

```diff
@@ -51,19 +50,19 @@ def install
     (lib/"pkgconfig/lua.pc").write pc_file
 
     # Fix some software potentially hunting for different pc names.
-    bin.install_symlink "lua" => "lua5.3"
-    bin.install_symlink "lua" => "lua-5.3"
-    bin.install_symlink "luac" => "luac5.3"
-    bin.install_symlink "luac" => "luac-5.3"
-    (include/"lua5.3").install_symlink Dir[include/"lua/*"]
-    lib.install_symlink "liblua.5.3.dylib" => "liblua5.3.dylib"
-    (lib/"pkgconfig").install_symlink "lua.pc" => "lua5.3.pc"
-    (lib/"pkgconfig").install_symlink "lua.pc" => "lua-5.3.pc"
+    bin.install_symlink "lua" => "lua5.4"
+    bin.install_symlink "lua" => "lua-5.4"
+    bin.install_symlink "luac" => "luac5.4"
+    bin.install_symlink "luac" => "luac-5.4"
+    (include/"lua5.4").install_symlink Dir[include/"lua/*"]
+    lib.install_symlink "liblua.5.4.dylib" => "liblua5.4.dylib"
+    (lib/"pkgconfig").install_symlink "lua.pc" => "lua5.4.pc"
+    (lib/"pkgconfig").install_symlink "lua.pc" => "lua-5.4.pc"
   end
 
   def pc_file
     <<~EOS
-      V= 5.3
+      V= 5.4
```

Comment:
```suggestion
      V= #{version.major_minor}
```

Reply:
Gotcha. Same for `lua@5.3.rb`, presumably?